### PR TITLE
Stop spamming our alumni friends

### PR DESCRIPTION
**Pull request by bricelam**
Also removing `ms.author` since the default is Arthur for our docset.

### CODE REVIEW

File: entity-framework/core/providers/writing-a-provider.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Writing a Database Provider - EF Core
 description: Information on writing a new Entity Framework Core provider
-author: anmiller
```
Comment (bricelam) [inline entity-framework/core/providers/writing-a-provider.md:4]: Must've been a relative of Rowan...

File: entity-framework/core/querying/related-data/index.md [hunk 2]
```diff
@@ -3,7 +3,6 @@ title: Loading Related Data - EF Core
 description: Different strategies for loading related data with Entity Framework Core
 author: roji
 ms.date: 9/11/2020
-ms.assetid: f9fb64e2-6699-4d70-a773-592918c04c19
```
Comment (bricelam) [inline entity-framework/core/querying/related-data/index.md:6]: ❓

File: entity-framework/ef6/fundamentals/async.md [hunk 3]
```diff
@@ -216,14 +216,14 @@ For a comprehensive list of available extension methods in the System.Data.Entit
 
 Now that the code is asynchronous, we can observe a different execution flow when we run the program:
 
-1. **SaveChanges** begins to push the new **Blog** to the database  
+1. **SaveChanges** begins to push the new **Blog** to the database
```
Comment (bricelam) [inline entity-framework/ef6/fundamentals/async.md:219]: Uh-oh. I'll check if these were significant

**Review by smitpatel — APPROVED** [review]

File: entity-framework/core/querying/related-data/index.md (hunk 2 above)
Comment (smitpatel) [reply entity-framework/core/querying/related-data/index.md:6]: asset id is not needed.

**Pull request merged**